src/pages/Progress.jsx

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+
+import './styles/Table.css';
+
+import { Hamburger } from '../components/Buttons/Hamburger/Hamburger';
+
+import startIcon from '../assets/images/startIcon.png';
+
+const memberName = 'Ivan';
+const pageName = 'Progress';
+
+const menuItems = ['Name', 'Start', 'Deadline', 'Actions'];
+const ProgressBody = [
+  {
+    name: 'Create the DB',
+    note: 'Implemented the TaskState table',
+    date: '28.01.2021',
+    date_img: startIcon,
+    buttons: 'buttons',
+  },
+  {
+    name: 'Create the DB',
+    note: 'Created the Member view',
+    date: '29.01.2021',
+    date_img: startIcon,
+    buttons: 'buttons',
+  },
+  {
+    name: 'Implement the props',
+    note: 'Implemented the calc progress proc',
+    date: '30.01.2021',
+    date_img: startIcon,
+    buttons: 'buttons',
+  },
+];
+
+export function Progress({ showDrawer, toggle }) {
+  return (
+    <article>
+      <header className='header'>
+        <Hamburger showDrawer={showDrawer} toggle={toggle} />
+      </header>
+      <p className='page-name'>
+        {`${memberName}'s ${pageName}`}
+        <span>{`(${ProgressBody.length})`}</span>
```

Comment:
```suggestion
     <span>{ProgressBody.length}</span>
```
it looks better

Reply:
Here i need brackets - (3), your proposal makes only number - 3

Reply:
```suggestion
     <span>({ProgressBody.length})</span>
```
it looks more simple

Reply:
Yeah, thats much accurate, thanks. Changed in your way